spec/DemographicDataset.schema.json

```diff
@@ -2,10 +2,13 @@
     "id": "DemographicDataSet",
     "type": "object",
     "properties": {
-        "code": { "$ref": "StandardIdentifier.schema.json" },
+        "id": { "$ref": "StandardIdentifier.schema.json" },
         "name": { "type": "string" },
+        "source": { "type": "string" },
+        "countries": { "type": "array", "items": { "$ref": "CountryId.schema.json"  } },
+        "variants": { "type": "array", "items": { "type": "string" } },
```

Comment:
this is weird - I actually removed variants from the spec, I don't know why the diff is still showing it